packages/wdio-spec-reporter/src/index.js

```diff
@@ -96,7 +96,9 @@ class SpecReporter extends WDIOReporter {
      */
     getTestLink ({ config, sessionId }) {
         if (config.hostname.includes('saucelabs')) {
-            const dc = ['eu', 'eu-central-1'].includes(config.region) ? '.eu-central-1' : ''
+            const dc = config.headless
+                ? '.us-east1.headless'
```

Comment:
headless can only be with 'us-east1' ?

Reply:
yes for now .. this might change as well but then I will make appropriate changes.